src/main/java/apoc/util/Util.java

```diff
@@ -700,4 +705,46 @@ public static boolean classExists(String className) {
         Object value = ApocConfiguration.get(loadType).get(key);
         return Optional.ofNullable(value).map(Object::toString);
     }
+
+    public static String formatDate(TemporalAccessor value, String format){
+        return getFormat(format).format(value);
+    }
+
+
+    public static TemporalAccessor parseDate(String value, Class<? extends TemporalAccessor> date, String...formats) {
+        String methodName = "parse";
+        try {
+            Method method = date.getDeclaredMethod(methodName, CharSequence.class, DateTimeFormatter.class);
+
+            for (String form : formats) {
+                try {
+                    try {
+                        return (TemporalAccessor) method.invoke(null, value, getFormat(form));
+                    } catch (InvocationTargetException e) {
+                        Method methodSimple = date.getDeclaredMethod(methodName, CharSequence.class);
```

Comment:
Adesso è diventato più costoso averlo qui dentro portarlo allo stesso livello di method...
Prima era meno costoso perché non c'era il `for`